.github/workflows/main.yml

```diff
@@ -44,7 +44,7 @@ jobs:
         uses: golangci/golangci-lint-action@v2
         with:
           version: v1.30
-          args: --timeout 5m
+          args: --timeout 5m --skip-dirs pkg/gen --skip-files "zz_generated.deepcopy.go$"
```

Comment:
Is the `$` at the intentional?

Reply:
Yes, `--skip-files` takes a regex. [Reference here](https://golangci-lint.run/usage/configuration/):

```
  # which files to skip: they will be analyzed, but issues from them
  # won't be reported. Default value is empty list, but there is
  # no need to include all autogenerated files, we confidently recognize
  # autogenerated files. If it's not please let us know.
  # "/" will be replaced by current OS file path separator to properly work
  # on Windows.
  skip-files:
    - ".*\\.my\\.go$"
    - lib/bad.go
```